### ISSUE TITLE
Focus outlines are not consistent and fail colour contrast

### ISSUE THREAD

**Issue by whitep4nth3r** [npmx-dev/npmx.dev#647]
In both light and dark mode, focus outlines when tabbing with a keyboard are not consistent and all fail colour contrast.

Screenshotting some examples:

<img width="578" height="212" alt="Image" src="https://github.com/user-attachments/assets/a4f38cd2-2b88-41dc-878b-c3c0b7e962ca" />

<img width="802" height="146" alt="Image" src="https://github.com/user-attachments/assets/181daaf0-ad83-46ca-afbe-66c31d0e4f1c" />

<img width="643" height="366" alt="Image" src="https://github.com/user-attachments/assets/5b3c7fa6-f189-43e2-948c-02245454b364" />

<img width="695" height="139" alt="Image" src="https://github.com/user-attachments/assets/5ed7f072-a18c-4469-8166-6fa3ff5a4dae" />

<img width="370" height="106" alt="Image" src="https://github.com/user-attachments/assets/eba8c707-b0a1-4896-8a90-d46e5c51d38b" />

<img width="385" height="114" alt="Image" src="https://github.com/user-attachments/assets/87d39514-981f-4727-93c8-4e256d2c5b0a" />
